lib/main_common.pm

```diff
@@ -2405,6 +2405,24 @@ sub load_security_tests_selinux {
     loadtest "security/selinux/chcat";
 }
 
+sub load_security_tests_cc {
+    # Setup environment for cc testing: 'audit-test' test suite setup
+    # Such as: download code branch; install needed packages
+    #loadtest 'security/cc/cc_audit_test_setup';
+
+    # Run test cases of 'audit-test' test suite which do NOT need SELinux env
+    #loadtest 'security/cc/audit_tools';
+
+    # Setup environment for cc testing: SELinux setup
+    # Such as: set up SELinux with permissive mode and specific policy type
+    #loadtest 'security/selinux/selinux_setup';
+    loadtest 'security/cc/cc_selinux_setup';
+
+    # Run test cases of 'audit-test' test suite which do need SELinux env
+    # Please add these test cases here: poo#93441
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details

Reply:
The openQA automation for "CC" testing is a team work 3 of us are woking on this atm. QE security members know what does this means so they can add their part accordingly without confusion or communication cost.

Reply:
Sorry for my misunderstanding. I know what you mean now, I was debugging.
Yes, I will add "WIP" next time.